main.js

```diff
@@ -43,6 +43,7 @@ module.exports = function(options) {
 		withNavigationHierarchy: false,
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
+		withoutAssetPreloading: false,
```

Comment:
a `withAssetUrlGeneration` option that deafults to true with a `// TODO` to make it default to `false` in the next major version sounds good 👍